.pre-commit-config.yaml

```diff
@@ -1,14 +1,14 @@
 repos:
       - repo: https://github.com/timothycrosley/isort
-        rev: 4.3.21
+        rev: 5.0.7
         hooks:
               - id: isort
       - repo: https://github.com/ambv/black
-        rev: stable
+        rev: 19.10b0
         hooks:
               - id: black
       - repo: https://gitlab.com/pycqa/flake8
-        rev: 3.7.7
+        rev: 3.8.3
```

Comment:
Sorry for the late review here, but why did we pin those versions specifically instead of those from the integration repo (see https://github.com/rapidsai/integration/blob/branch-0.18/conda/recipes/versions.yaml) ?

Reply:
I think some of these tools weren't respecting line length configuration values in some cases. Ran into similar issues as well. Though this does a raise a good point, would be good to propagate these version bumps into the integration repo.

Reply:
Ideas on what's the right approach? Pinning those exact versions in integration will probably lead all other RAPIDS repos to require some reformatting.